.github/workflows/test.yml

```diff
@@ -1,70 +1,69 @@
-    - name: setup dependencies
-      run: |
-       sudo apt update -y
-       sudo apt-get -y install lua5.1 luarocks
-       sudo luarocks install luacheck
-    - name: set default pip to pip3
-      run: |
-       sudo rm -f /usr/local/bin/pip
-       sudo ln -s /usr/bin/pip3 /usr/local/bin/pip
-    - name: set default python and pip to v2.7
-      if: ${{ matrix.python-version }} == 2.7
-      run: |
-       curl https://bootstrap.pypa.io/pip/2.7/get-pip.py --output get-pip.py
-       sudo python2 get-pip.py
-       sudo rm -f /usr/bin/python
-       sudo ln -s /usr/bin/python2 /usr/bin/python
-       sudo rm -f /usr/local/bin/pip
-       sudo ln -s /usr/local/bin/pip2 /usr/local/bin/pip
-    - name: setup python dependencies
-      run: |
-       pip install -r requirements.txt
-       pip install -r requirements-test.txt
-    - name: run static analysis
-      run: |
-       make lint
-    - name: run regression tests
-      run: |
-       make test
-    - name: code coverage
-      if: ${{ matrix.python-version == '3.8' && matrix.tarantool-version == '2.7' }}
-      run: |
-       pip install coveralls==3.*
-       make coverage
-    - name: upload coverage data to coveralls.io
-      if: ${{ matrix.python-version == '3.8' && matrix.tarantool-version == '2.7' }}
-      run: coveralls --service=github
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: recursive
+
+      - name: set up Tarantool ${{ matrix.tarantool-version }}
+        uses: tarantool/setup-tarantool@v1
+        with:
+          tarantool-version: ${{ matrix.tarantool-version }}
+
+      - name: set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: display Python version
+        run: python -c "import sys; print(sys.version)"
+
+      - name: setup dependencies
+        run: |
+          sudo apt update -y
+          sudo apt-get -y install lua5.1 luarocks
+          sudo luarocks install luacheck
+
+      - name: setup Python dependencies
+        run: |
+          python -m pip install -r requirements.txt
+          python -m pip install -r requirements-test.txt
+
+      - name: run static analysis
+        run: make lint
+
+      - name: run regression tests
+        run: |
+          sudo rm /usr/bin/python
+          make test
+
+      - name: code coverage
+        if: ${{ matrix.python-version == '3.8' &&
+          matrix.tarantool-version == '2.7' }}
+        run: |
+          python -m pip install coveralls==3.*
+          make coverage
+
+      - name: upload coverage data to coveralls.io
+        if: ${{ matrix.python-version == '3.8' &&
+          matrix.tarantool-version == '2.7' }}
+        run: coveralls --service=github
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why do you need to fix indentation here? Commit message describes changes as "refactoring", however changes includes adding spaces and changing indentation. I would fix only a problem `${{ matrix.python-version }} == 2.7` and revert other changes as useless.

Reply:
Formatting changes should not make it hard to read changes of a visible behaviour. Please, discard or move to its own commit.